src/app/cli/src/output/output_config.rs

```diff
@@ -26,6 +26,18 @@ pub struct OutputConfig {
     pub trace_file: Option<PathBuf>,
 }
 
+impl Default for OutputConfig {
+    fn default() -> Self {
+        Self {
+            quiet: false,
+            verbosity_level: 0,
+            is_tty: false,
```

Comment:
Q: should we use true as the default value?

Reply:
While I can see that this is not a new change, but a legacy -- still, it is interesting to ask

Reply:
`OutputConfig::default()` is currently used in just one test, and we probably never want to have fancy TTY output in tests.